test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -74,11 +74,19 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
```

Comment:
unused import?

Reply:
thanks.  I dropped the fixed method in favor of resetting the seed and didn't update the imports